docs/release-notes.md

```diff
@@ -1,157 +1,101 @@
 
-In an effort to continuously improve Open AMT Cloud Toolkit, the team overhauled the activation and configuration flows inside of RPS. Utilizing [xstate](https://xstate.js.org/), we modularized each of the configuration actions so they can be called as needed by the state machine. This will add increased robustness to complex configuration flows as well as make it easier to add new configuration options in the future.
+We've had several customer requests to add this feature to MPS and this release we are delivering.  We've added an API end point to MPS that will send the unprovision command to AMT and will then remove that AMT device from the MPS database.  In order to reconnect that AMT device to MPS again, RPC and RPS will need to be used.
 
-:material-new-box:** Customer Request: Set AMT Features During Configuration **
+:material-star:** Customer Request: Enable local wifi profile synchronization by default **
 
-We have updated the AMT Profile API to include options for setting Redirection, User Consent, SOL, IDE-R, and KVM. Setting these in the profile will override the default AMT settings with the preferred settings from the profile at configuration time.
-
-:material-new-box:** Improvement: Certificate Authentication in Domain Profile**
-
-When creating a domain profile, we now validate that the password provided for the provisioning certificate works for the provided provisioning certificate. This allows users to fix an incorrect password entry instead of getting a failure during activation.
-
-:material-new-box:** Improvement: RPC Quality of Life Improvements**
-
-We have added several quality of life improvements to RPC. AMTINFO command now displays only the network adapter information that is available in the system instead of showing wireless if wireless is not present (and vice versa). Added an admin privilege check and AMT not found check. Added warnings if link status is down or dns suffix is not populated. We replaced our log.Fatal with log.Error and have replaced our os.Exit calls on error to allow better usability when RPC is compiled as a library.
-
-:material-star:** Coming Soon: Deactivate AMT without deploying RPC and new Maintenance Options**
-
-In our next release, we are enhancing our MPS delete devices API to deactivate AMT as well as removing it from the MPS database. This is a customer feature request to help with decommissioning or transferring devices out of service. We will also be adding additional AMT Maintenance commands to RPC that will help keep AMT updated and running smooth.
+We have updated RPS to enable AMT to sync wifi profiles automatically if a local management service (LMS) is running.  This means if a user adds a wireless profile to the system, this wireless profile will automatically be added to AMT and, when the OS is down, AMT will use this wireless profile to connect to the network so that the device can be remotely managed.
```

Comment:
This means if a user connects to a wireless network in the OS, this wireless network will automatically be synced to AMT. When the OS is down, AMT will use this wireless network to connect to the network so that the device can be remotely managed.